openedx/core/djangoapps/user_authn/api/helper.py

```diff
@@ -84,25 +86,30 @@ def __init__(self, field_type='required'):
                 ):
                     self.valid_fields.append(field_name)
 
+    def _field_exist_in_UserProfile_fields_or_user_extended_config(self, field):
```

Comment:
Add a docstring to explain what you are checking

```suggestion
    def _field_cannot_be_saved(self, field):
```